### ISSUE TITLE
Message component text in dark mode does not meet accessibility requirements

### ISSUE THREAD

**Issue by dcb99** [primefaces/primevue-tailwind#364]
As an example:
<img width="650" alt="image" src="https://github.com/user-attachments/assets/07828ab3-1193-44eb-b49e-4d5300d834e3">
Blue text on a blue background doesn't provide enough contrast and is a poor UX, especially for those with color blindness.

**Comment by tugcekucukoglu** [comment]
Please create a ticket for PrimeVue. When it is implemented in PrimeVue, it will be ported to primevue-tailwind.

Thank you!
